starkit/tests/test_spectral.py

```diff
@@ -0,0 +1,6 @@
+from starkit.utils import spectral
+
+def test_fwhm2sigma():
+    assert spectral.fwhm2sigma(spectral.FWHM2SIGMA_CONST) == 1.0
```

Comment:
This is a smart way of doing it!

Reply:
Do you suggest any changes here? Or can it be merged?